eml/components/explore/FilterNavBar.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import SearchBar from './SearchBar'; 
+import { View, Text, Pressable } from 'react-native';
+import { ScrollView } from 'react-native-gesture-handler';
+
+const categories = [
```

Comment:
Maybe put this in a seperate file if intended to use in production

Reply:
I agree